libethereum/ClientBase.cpp

```diff
@@ -417,9 +417,12 @@ BlockHeader ClientBase::pendingInfo() const
 
 BlockDetails ClientBase::pendingDetails() const
 {
-    auto pm = postSeal().info();
-    auto li = Interface::blockDetails(LatestBlock);
-    return BlockDetails((unsigned)pm.number(), li.totalDifficulty + pm.difficulty(), pm.parentHash(), h256s{});
+    auto pendingHeader = postSeal().info();
+    auto latestDetails = Interface::blockDetails(LatestBlock);
+    return BlockDetails
+    {static_cast<unsigned>(pendingHeader.number()),
```

Comment:
weird indentation, is this clang-format?

Reply:
@gumb0 Yes, clang did that...i can manually format it a little nicer